src/activities/assignments/AssignmentEntity.js

```diff
@@ -780,6 +798,12 @@ export class AssignmentEntity extends Entity {
 			}
 		}
 
+		if (typeof assignment.defaultScoringRubricId !== undefined &&
```

Comment:
should undefined be 'undefined' like above?

Reply:
Ah good catch, yes it should